src/org/mangui/hls/flv/FLVTag.as

```diff
@@ -20,6 +20,8 @@ package org.mangui.hls.flv {
         public static const DISCONTINUITY : int = 5;
         /** metadata Type ID. **/
         public static const METADATA : int = 6;
+        /** captiondata Type ID. **/
+        public static const CAPTIONDATA : int = 42;
```

Comment:
Minor, but any specific reason for 42 instead of going to the next sequential number? And maybe CAPTION_DATA instead of CAPTIONDATA.

Reply:
I was going to do that, but I was a bit concerned that it goes 1-6 then 8-9 then 18, I wasn't sure why 7 was skipped, and didn't want to somehow conflict with something else in the code so I just went with something completely different. I can change it to 7 if you want, shouldn't make much difference either way.

Reply:
Not too much difference, and the concern makes sense. Maybe go three digits though (at least that can identify, for our purposes, ones we've added)? It's pretty arbitrary, but at least that should prevent ever conflicting (though I doubt we will ever conflict), and sort it to the end of the list.